prime-router/src/main/kotlin/fhirengine/translation/hl7/schema/ConfigSchema.kt

```diff
@@ -161,6 +161,10 @@ abstract class ConfigSchemaElement(
 ) {
     private var validationErrors: MutableSet<String> = mutableSetOf()
 
+    override fun toString(): String {
```

Comment:
I just moved this up to the abstract class so both implementations will work the same way.

Reply:
I think you're right... I'll just drop it and we can revisit it later.  The exception will now capture the enclosing schema.